static/app/components/events/eventTagsAndScreenshot/screenshot/screenshotPagination.tsx

```diff
@@ -0,0 +1,72 @@
+// eslint-disable-next-line no-restricted-imports
+import {withRouter, WithRouterProps} from 'react-router';
+import styled from '@emotion/styled';
+
+import Button from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import {CursorHandler} from 'sentry/components/pagination';
+import {IconChevron} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import parseLinkHeader from 'sentry/utils/parseLinkHeader';
+
+type Props = WithRouterProps & {
+  nextDisabled: boolean;
+  onCursor: CursorHandler;
+  previousDisabled: boolean;
+  pageLinks?: string | null;
+};
+
+/*
+  HACK: Slight variation of the Pagination
+  component that allows the parent to control
+  enabling/disabling the pagination buttons.
+*/
+const ScreenshotPagination = ({
+  location,
+  onCursor,
+  pageLinks,
+  previousDisabled,
+  nextDisabled,
+}: Props) => {
+  if (!pageLinks) {
+    return null;
+  }
+
+  const path = location.pathname;
+  const query = location.query;
+  const links = parseLinkHeader(pageLinks);
+
+  return (
+    <Wrapper>
+      <ButtonBar merged>
+        <Button
+          icon={<IconChevron direction="left" size="sm" />}
+          aria-label={t('Previous')}
+          size="md"
+          disabled={previousDisabled}
+          onClick={() => {
+            onCursor?.(links.previous?.cursor, path, query, -1);
+          }}
+        />
+        <Button
+          icon={<IconChevron direction="right" size="sm" />}
+          aria-label={t('Next')}
+          size="md"
+          disabled={nextDisabled}
+          onClick={() => {
+            onCursor?.(links.next?.cursor, path, query, 1);
+          }}
```

Comment:
I was wondering if it would be worthwhile to move these `onClick`s up and have an `onPrevious` and `onNext`, might be useful for analytics things but this might be a YAGNI situation.

I just felt like expecting the `onCursor` to be ready to accept a delta was too specific. I think you'd have to lift up some of the data to get `path`, `query`, and `links` but maybe we have those available in the parent already? Not sure. If there are any issues with this suggestion feel free to ignore it because I think you've clearly denoted this code fits a specific use-case for now